apps/financial-aid/web-osk/src/routes/status/MainPage/mainPage.tsx

```diff
@@ -26,29 +26,49 @@ import { AppContext } from '@island.is/financial-aid-web/osk/src/components/AppP
 const MainPage = () => {
   const logOut = useLogOut()
 
-  const { myApplication, loading, error, municipality } = useContext(AppContext)
+  const {
+    myApplication,
+    loading,
+    error,
+    municipality,
+    setMunicipality,
+    user,
+  } = useContext(AppContext)
+
+  const isUserSpouse = user?.isSpouse?.HasApplied
```

Comment:
Is this check correct?

Reply:
Yess, would you rather check if spouse has files, or maybe both, to be thorough?

Reply:
No, just the naming confused me because I read this is has spouse applied and then it is extracted into a variable called isUserSpouse 🙈 
Which is bascially why I would rather go the way mentioned above 😄